analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ImplementISerializableCorrectly.CSharp9.cs

```diff
@@ -170,33 +170,44 @@ public new void GetObjectData(SerializationInfo info, StreamingContext context)
     }
 
     [Serializable]
-    public partial record Partial_SerializableDerived_CallingBase_GetObjectData_Record : SerializableRecord
+    public record Serializable_NoConstructor_Positional(string Value) : ISerializable
+//                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Noncompliant {{Update this implementation of 'ISerializable' to conform to the recommended serialization pattern.}}
+//         ^^^^^^ Secondary@-1 {{Add a 'protected' constructor 'Serializable_NoConstructor_Positional(SerializationInfo, StreamingContext)'.}}
+    {
+        public Serializable_NoConstructor_Positional() : this("") { }
+        public virtual void GetObjectData(SerializationInfo info, StreamingContext context) { }
+    }
+
+// See https://github.com/SonarSource/sonar-dotnet/issues/4411 Applicable to all raised issues below
```

Comment:
Put the cases in a separate namespace. This way if somebody adds a non related case below it can create confusion. And also please add at least one test case with classes.

Reply:
Actually, last case is the case with classes, it also doesn't work for the same reason - partial methods are not supported. Added comment in it back and extracted everything in another namespace.

Reply:
I missed the fact that the last was a class. Thank you anyway for putting them into a namespace.